### PR TITLE
packaging: fix sev kernel caching

### PR DESCRIPTION
**Pull request by UnmeshDeodhar**
Check if tarball exists. If yes, skip the tarball create command.

Fixes: #6473

### CODE REVIEW

**Review by snir911 — APPROVED** [review]
LGTM thanks

File: tools/packaging/static-build/cache_components.sh [hunk 1]
```diff
@@ -48,7 +48,9 @@ cache_kernel_artifacts() {
 
 	if [ "${TEE}" == "sev" ]; then
 		module_dir="${repo_root_dir}/tools/packaging/kata-deploy/local-build/build/cc-sev-kernel/builddir/kata-linux-${current_kernel_version#v}-$(get_config_version)/lib/modules/${current_kernel_version#v}"
-		tar cvfJ "${repo_root_dir}/tools/packaging/kata-deploy/local-build/build/kata-static-cc-sev-kernel-modules.tar.xz" "${module_dir}/kernel/drivers/virt/coco/efi_secret/"
+		if [ ! -f "${repo_root_dir}/tools/packaging/kata-deploy/local-build/build/kata-static-cc-sev-kernel-modules.tar.xz" ]; then
```
Comment (jepio) [inline tools/packaging/static-build/cache_components.sh:51]: lets use a local variable for this path, it's pretty long

**Review by jepio — APPROVED** [review]

**Comment by UnmeshDeodhar** [comment]
@fidencio, all checks are passing except TDX. And a static check that I think is a known issue.

**Pull request merged**